### PR TITLE
Difference of Squares - Added Parameters to placeholders

### LINKED ISSUE #555: difference-of-squares: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by Sukhj1nder**
Added Parameters to the Difference of Squares file in the functions `square_of_sum`, `sum_of_squares` and `difference`.

This resolves the issue #585.
TODO:  Difference of Squares - Ticked on #509

Resolves #555

### CODE REVIEW

File: exercises/difference-of-squares/difference_of_squares.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
     pass
 
 
-def sum_of_squares():
+def sum_of_squares(sum_of_squares):
     pass
 
 
-def difference():
+def difference(square_of_sum, sum_of_squares):
```
Comment (m-a-ge) [inline exercises/difference-of-squares/difference_of_squares.py]: `difference` takes only one parameter according to https://github.com/exercism/python/blob/master/exercises/difference-of-squares/difference_of_squares_test.py

File: exercises/difference-of-squares/difference_of_squares.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
-def square_of_sum():
+def square_of_sum(square_of_sum):
```
Comment (m-a-ge) [inline exercises/difference-of-squares/difference_of_squares.py]: This conflicts with the exercise itself, it requires only number

**Comment by Sukhj1nder** [comment]
@m-a-ge  cool I've made the changes. Does it look like what you wanted?

**Comment by Sukhj1nder** [comment]
@m-a-ge are these the changes you wanted?

**Comment by m-a-ge** [comment]
@Sukhj1nder No, not really. 
Natural number is expected in the exercise. `square_of_sum` is confusing

**Comment by Sukhj1nder** [comment]
@m-a-ge hopefully this is better

**Comment by cmccandless** [comment]
@Sukhj1nder Are you still working on this?

**Comment by Sukhj1nder** [comment]
Sorry guys, I have gotten busier with some of my other projects. Hopefully it gets finished but, I hope to see you guys in the future when I've finished my projects so I can contribute further into this project. 
Cheers Sukhi

**Pull request closed without merging**